cluster-up/cluster/kind/migration.sh

```diff
@@ -0,0 +1,38 @@
+ #!/usr/bin/env bash
+
+set -xe
+
+source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+
+function kind_migration::enable_pod_preset_feature() {
+    local -r cluster_name=$1
+
+    docker exec "$cluster_name-control-plane" bash -c 'sed -i \
+    -e "s/NodeRestriction/NodeRestriction,PodPreset/" \
+    -e "/NodeRestriction,PodPreset/ a\    - --runtime-config=settings.k8s.io/v1alpha1=true" \
+    /etc/kubernetes/manifests/kube-apiserver.yaml'
+}
+
+function kind_migration::create_virt_launcher_fake_product_uuid_podpreset() {
+    local -r namespace=$1
+
+    if ! _kubectl get ns "$namespace" &>2; then
+        _kubectl create ns "$namespace"
+    fi
+
+    _kubectl apply -f "$KIND_MANIFESTS_DIR/product-uuid-podpreset.yaml" -n "$namespace"
+}
+
+function kind_migration::enable_live_migration() {
```

Comment:
I would have called it per what it does, not per what it serves for.
Something like: "expose_unique_product_uuid_per_node".

How it is done is abstracted inside.
Why it is done, is explained at the caller that wants to use this functionality.

This also means this script name should change to "podpreset.sh".

Reply:
My intention was to call what it does w/o explaining mentioning how as we plan to replace the podpreset with a webhook, its the implementation of we are going to patch virt-launcher pods.

Looking at this again what you suggests make sense, 
I wil rename it to podPreset and move `kind_migration::enable_live_migration` to the caller (provider.sh)